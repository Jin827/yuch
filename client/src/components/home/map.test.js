import React from 'react';
import { shallow, mount } from 'enzyme';
import Map from './map';

describe('<Map />', () => {
  const wrapper = shallow(<Map />);

  it('renders properly', () => {
    expect(wrapper).toMatchSnapshot();
  });

  describe('when mounted', () => {});
});
