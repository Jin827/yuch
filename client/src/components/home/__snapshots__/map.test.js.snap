// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Map /> renders properly 1`] = `
<div
  id="map"
/>
`;
